src/Widgets/SearchToolbar.vala

```diff
@@ -49,13 +49,23 @@ namespace Terminal.Widgets {
                 _("Next result")
             );
 
-            cycle_button = new Gtk.ToggleButton ();
-            cycle_button.image = new Gtk.Image.from_icon_name (
-                "media-playlist-repeat-symbolic", Gtk.IconSize.SMALL_TOOLBAR
-            );
-            cycle_button.sensitive = false;
-            cycle_button.set_can_focus (false);
-            cycle_button.tooltip_text = _("Cyclic search");
+            cycle_button = new Gtk.ToggleButton () {
+                active = false,
+                can_focus = false,
+                image = new Gtk.Image ()
+            };
+            cycle_button.toggled.connect (() => {
+                if (cycle_button.active) {
+                    cycle_button.tooltip_text = _("Cyclic search is enabled");
```

Comment:
Tooltip text for image buttons should describe the action that will happen when the button is pressed

Reply:
Done